### PR TITLE
BUGFIX: improve documentation of ckEditor customization

### PR DESCRIPTION
**Pull request by skurfuerst**

### CODE REVIEW

File: packages/neos-ui-ckeditor5-bindings/src/manifest.config.js [hunk 1]
```diff
@@ -52,13 +52,30 @@ export default ckEditorRegistry => {
...
         This registry allows to register a custom configuration processor that takes a configuration object, modifies it and returns a new one. Example:
 
-        config.set('doSmthWithConfig' ckeConfig => {
+        config.set('doSmthWithConfig', ckeConfig => {
             ckeConfig.mySetting = true;
             return ckeConfig;
-        })
+        });
+        
+        The callback function actually gets passed TWO parameters; aside of the ckeConfig as first parameter, an object
```
Comment (dimaip) [inline packages/neos-ui-ckeditor5-bindings/src/manifest.config.js]:
```suggestion
        The callback function gets passed TWO parameters; aside of the ckeConfig as first parameter, an object
```

File: packages/neos-ui-ckeditor5-bindings/src/manifest.config.js [hunk 2]
```diff
@@ -52,13 +52,30 @@ export default ckEditorRegistry => {
...
+        config.set('doSmthWithConfig', (ckeConfig, {editorOptions}) => {
+            if ($get(['formatting', 'myCustomField'], editorOptions)) {
+                ckeConfig.mySetting = true;
+            }
+            return ckeConfig;
+        });
 
         That is all you need to know about configuring CKE in Neos,
-        refer to CKeditor5 documentationfor more details on what you can do with it: https://docs.ckeditor.com/ckeditor5/latest/index.html
+        refer to CKeditor5 documentation for more details on what you can do with it: https://docs.ckeditor.com/ckeditor5/latest/index.html
```
Comment (dimaip) [inline packages/neos-ui-ckeditor5-bindings/src/manifest.config.js]:
```suggestion
        refer to CKEditor 5 documentation for more details on what you can do with it: https://docs.ckeditor.com/ckeditor5/latest/index.html
```

Comment (markusguenther) [reply packages/neos-ui-ckeditor5-bindings/src/manifest.config.js]: Adjusted that :)

File: packages/neos-ui-ckeditor5-bindings/src/manifest.config.js (hunk 1 above)
Comment (markusguenther) [reply packages/neos-ui-ckeditor5-bindings/src/manifest.config.js]: Adjusted that part.

**Pull request merged**